relentless/simulate/lammps.py

```diff
@@ -445,14 +450,17 @@ class MinimizeEnergy(LAMMPSOperation):
         Force convergence criterion.
     max_iterations : int
         Maximum number of iterations to run the minimization.
-    dt : float
-        Maximum step size.
+    options : dict
+        Additional options for energy minimzer.
 
     """
-    def __init__(self, energy_tolerance, force_tolerance, max_iterations, dt):
+    def __init__(self, energy_tolerance, force_tolerance, max_iterations, options):
         self.energy_tolerance = energy_tolerance
         self.force_tolerance = force_tolerance
         self.max_iterations = max_iterations
+        self.options = options
+        if 'max_evaluations' not in self.options:
+            self.options['max_evaluations'] = 100*self.max_iterations
```

Comment:
Small issue: this default will not update if `max_iterations` is changed after the operation is made. I suggest setting this to `None` here, then in the `to_commands` method compute `maxeval` if `self.options['max_evaluations'] is None`.